mondrian/src/main/java/mondrian/web/servlet/MdxQueryServlet.java

```diff
@@ -21,9 +21,9 @@
 import java.io.IOException;
 import java.util.Enumeration;
 import java.util.List;
-import javax.servlet.ServletConfig;
-import javax.servlet.ServletException;
-import javax.servlet.http.*;
+import jakarta.servlet.ServletConfig;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.*;
```

Comment:
Import the specific classes instead of *. I know it was already there before but, just to follow the convention